ci/cases/gfsv17/ocnanal.yaml

```diff
@@ -0,0 +1,35 @@
+defaults:
+  !INC {{ HOMEgfs }}/parm/config/gfs/yaml/defaults.yaml
+
+base:
+  DOIAU: "YES"
+  DO_JEDIATMVAR: "NO"
+  DO_JEDIATMENS: "NO"
+  DO_JEDIOCNVAR: "YES"
+  DO_JEDISNOWDA: "NO"
+  DO_MERGENSST: "NO"
+  DOHYBVAR: "NO"
+  DO_FIT2OBS: "YES"
+  DO_VERFOZN: "YES"
+  DO_VERFRAD: "YES"
+  DO_VRFY_OCEANDA: "NO"
+  FHMAX_GFS: 240
+  HPSS_PROJECT: {{ 'HPSS_PROJECT' | getenv }}
+  HOMEDIR: {{ 'PWD' | getenv }}/HOMEDIR
+  STMP: {{ 'PWD' | getenv }}/TMP 
+  PTMP: {{ 'PWD' | getenv }}/TMP
+  NOSCRUB: {{ 'PWD' | getenv }}/TMP
```

Comment:
Not sure why `PWD` is here.

Reply:
Because stmp is always full.... I can remove this from this test though.   It might be nice in the future to have like a "user" type yaml that updates all the configs that a person might need to do, I don't know how that would work so this is just spit-balling an idea here.

Reply:
Yes.  `stmp` being full is a problem. 
The feature request is noted. We will do what we can.

Reply:
maybe replace with this @JessicaMeixner-NOAA :
```
{{ 'RUNTESTS' | getenv }}
```
for stmp and ptmp ?